wire/message.go

```diff
@@ -303,24 +302,30 @@ func WriteMessageN(w io.Writer, msg Message, pver uint32, dcrnet CurrencyNet) (i
 	cmd := msg.Command()
 	if len(cmd) > CommandSize {
 		msg := fmt.Sprintf("command [%s] is too long [max %v]", cmd, CommandSize)
-		return totalBytes, messageError(op, ErrCmdTooLong, msg)
+		return 0, messageError(op, ErrCmdTooLong, msg)
 	}
 	copy(elems.command[:], []byte(cmd))
 
+	// Initialize buffer with zeroed bytes for the message header (to be
+	// filled in, with checksum, after appending the payload
+	// serialization).
+	buf := new(bytes.Buffer)
```

Comment:
Please keep the existing style.  Using `new` like this is inconsistent with the rest of the code.  Similarly, the code generally uses `buf` for byte slices.  This is being used as a writer, so please just use `var bw bytes.Buffer` like it already was.

I know it might not matter in terms of functionality, but I want things to be consistent since I usually end up doing the vast majority of the maintenance and I, frankly, despise this style.